Makefile

```diff
@@ -54,6 +51,11 @@ DOCKER_IMAGE_DOCKERLINT += "hadolint/hadolint:v2.9.2@sha256:d355bd7df747a0f124f3
 DOCKER_IMAGE_COSIGN     += "bitnami/cosign:1.8.0@sha256:8c2c61c546258fffff18b47bb82a65af6142007306b737129a7bd5429d53629a"
 DOCKER_IMAGE_GH_CLI     += "registry.internal.mattermost.com/images/build-ci:3.16.0@sha256:f6a229a9ababef3c483f237805ee4c3dbfb63f5de4fbbf58f4c4b6ed8fcd34b6"
 
+DOCKER_IMAGE_RUNNER     := "debian:sid-20230109@sha256:c7caaec69b8b2d56332fb0860252e7865f6900f8031c845ae8f5f16045c619bd"
+ifeq ($(ARCH),arm64)
+	DOCKER_IMAGE_RUNNER="arm64v8/debian:sid-20230109@sha256:518e58d25b6ec2ed341247fbcd0c1cd992de98b8323568164a588ec34f588aec"
+endif
+
```

Comment:
As we want the hashes I don't think we can optimize much more, so moved them here for simplicity.